src/cmd/services/m3query/scripts/curlDiff.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script can be used to compare prometheus query output with m3query output
+# Sample usage: bash curlDiff.sh "count_over_time(go_gc_duration_seconds[30s])" 1535125074
+target=${1?"Missing target, usage: $0 target start"}
+start=${2?"Missing start, usage: $0 target start"}
+duration=${3:1000}
+end=$((start + duration))
+m3command="localhost:7201/api/v1/prom/native/read?start=$start&end=$end&step=15s&debug=true --data-urlencode target=$target"
```

Comment:
Take m3 endpoint and prom endpoint as parameters

Reply:
don't want to make the script too complicated. We can definitely add that when needed